src/js/view/controls/controlbar.js

```diff
@@ -257,8 +257,10 @@ export default class Controlbar {
         _model.on('change:captionsList', onCaptionsChanged, this);
         _model.change('nextUp', (model, nextUp) => {
             let tipText = localization.nextUp;
-            if (nextUp) {
+            if (nextUp && nextUp.title) {
                 tipText = (`NEXT: ${nextUp.title}`);
+            } else {
+                tipText = (`NEXT UP`);
```

Comment:
Remove the else, L259 handles the default case with localization.

Reply:
👍 
=